### PR TITLE
Feature/collapsible section

### PR DESCRIPTION
**Pull request by tyler-hoffman**

### CODE REVIEW

File: react-native/components/src/collapsible-section/collapsible-section.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+  /** Whether or not the section starts open - defaults to true */
+  startOpen?: boolean;
+}
+
+interface CollapsibleSectionState {
+  collapsed: boolean;
+}
+
+export class CollapsibleSection extends Component<CollapsibleSectionProps, CollapsibleSectionState> {
```
Comment (patgeek) [inline react-native/components/src/collapsible-section/collapsible-section.tsx]: Add component description above here

File: react-native/components/src/collapsible-section/collapsible-section.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          testID={'header'}
+          style={{ flexDirection: 'row' }}
+          onPress={() => this.setState({ collapsed: !collapsed })}
+        >
+          <View style={[styles.headerContainer, styles.withBackground]}>
+            <View style={[styles.headerRow, styles.withMargin]}>
+              <Text style={styles.title}>{title}</Text>
+              <MaterialIcon
+                style={[collapsed ? styles.iconImageUp : styles.iconImageDown]}
+                size={24}
```
Comment (patgeek) [inline react-native/components/src/collapsible-section/collapsible-section.tsx]: Make this and colors into `static readonly` variables at top of class

File: react-native/components/src/collapsible-section/collapsible-section.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { StyleProp, View, TouchableOpacity, Text, StyleSheet } from 'react-native';
+import MaterialIcon from 'react-native-vector-icons/MaterialCommunityIcons';
+import Collapsible from 'react-native-collapsible';
+import { gray, blue } from '@pxblue/colors';
+
+export interface CollapsibleSectionProps {
```
Comment (patgeek) [inline react-native/components/src/collapsible-section/collapsible-section.tsx]: Add some basic style props to this like `backgroundColor`, `iconColor`, etc.

Comment (patgeek) [reply react-native/components/src/collapsible-section/collapsible-section.tsx]: Also maybe a `disabled` prop

File: react-native/docs/collapsible-section.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+<CollapsibleSection title={'Section Title Here'}>
+  <View style={{ }}>
+    <Text>Your content here!</Text>
+  </View>
+</CollapsibleSection>
+```
+
+Note: Child components will be the content to collapse/expand
+
+## Props
```
Comment (patgeek) [inline react-native/docs/collapsible-section.md]: Don't forget to update this after adding props

File: react-native/components/src/collapsible-section/collapsible-section.tsx (hunk 3 above)
Comment (tyler-hoffman) [reply react-native/components/src/collapsible-section/collapsible-section.tsx]: ok

**Pull request merged**